students/jemielity_kamil/lesson_05_lists/swap_the_columns.py

```diff
@@ -0,0 +1,24 @@
+def swap_columns(a, i, j):
+
+    for loop in range(len(a)):
+        a[loop][i], a[loop][j] = a[loop][j], a[loop][i]
+
+    for i in range(len(a)):
+        for j in range(len(a[0])):
+            a[i][j] = str(a[i][j])
+
```

Comment:
Other than in C, Python doesn't like a new line in front of 'return' or 'for' statements. It's not a mistake but rather a good habit.